hardhat.config.ts

```diff
@@ -103,4 +104,12 @@ task('remove-tokens', 'Removes a list of tokens')
   }
 );
 
+task('deploy-collector', 'Deploys the collector')
+  .addOptionalParam('collectorConfigFileName', 'Path to an alternative config file')
+  .addOptionalParam('outputFileName', 'Path to the output file')
+  .setAction(async (taskArgs: DeployCollectorArg, hre) => {
+    await deployCollector(taskArgs, hre);
+  }
+);
```

Comment:
I would consider generalising this into a single deploy task. this way we could utilise the hardhat cli powers.

Reply:
I have thought about this as well, but I think it falls out of scope for what this task requires.

Reply:
I've actually tried to use the subtasks, but it looks like they're particularly useful when we want to split a task into multiple actions. For our specific use case, I think it complicates the code without any benefit.